.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,70 @@
 
-### Tests
+Opening GitHub issues ahead of time contributes to the [Openness](http://theapacheway.com/open/#:~:text=Openness%20allows%20new%20users%20the,must%20happen%20in%20the%20open.) of the Apache Parquet-Java project.
 
-- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
+Then could you also rename the pull request title in the following format?
 
-### Commits
+    GH-${GITHUB_ISSUE_ID}: ${SUMMARY}
 
-- [ ] My commits all reference GitHub issues in their subject lines. In addition, my commits follow the guidelines
-  from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
-    1. Subject is separated from body by a blank line
-    1. Subject is limited to 50 characters (not including GitHub issue reference)
-    1. Subject does not end with a period
-    1. Subject uses the imperative mood ("add", not "adding")
-    1. Body wraps at 72 characters
-    1. Body explains "what" and "why", not "how"
+or
 
-### Style
-- [ ] My contribution adheres to the code style guidelines and Spotless passes.
-    - To apply the necessary changes, run `mvn spotless:apply -Pvector-plugins`
+    MINOR: ${SUMMARY}
 
-### Documentation
+-->
 
-- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
-    - All the public functions and the classes in the PR contain Javadoc that explain what it does
+### Rationale for this change
+
+<!--
+ Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
+ Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.  
+-->
+
+### What changes are included in this PR?
+
+<!--
+There is no need to duplicate the description in the issue here but it is sometimes worth providing a summary of the individual changes in this PR.
+-->
+
+### Are these changes tested?
+
+<!--
+We typically require tests for all PRs in order to:
+1. Prevent the code from being accidentally broken by subsequent changes
+2. Serve as another way to document the expected behavior of the code
+
+If tests are not included in your PR, please explain why (for example, are they covered by existing tests)?
+-->
+
+### Are there any user-facing changes?
+
+<!--
+If there are user-facing changes then we may require documentation to be updated before approving the PR.
+-->
+
+<!--
+Please uncomment the line below (and provide explanation) if you are adding a dependency. Check if the license complies with
+the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
+-->
+<!-- **This PR adds new dependency.** -->
+
+<!--
+If there are any breaking changes to public APIs, please uncomment the line below and explain which changes are breaking.
+-->
+<!-- **This PR includes breaking changes to public APIs.** -->
+
+<!--
```

Comment:
I might want to leave the critical fix out here. I dislike long PR templates, and I find that people don't really read them :'( Less is more I would say, or we should link to existing content (for example, link to the ASF Security page: https://www.apache.org/security/).